pkg/arrai/integration_diagram.md

```diff
@@ -0,0 +1,18 @@
+# Integration Diagram
+
```

Comment:
Maybe embed the generated SVG here?

Reply:
It is, but GitHub thinks it's broken. Let's see after submit.